.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+  - "2.7"
+install:
+  - pip install ansible==2.2.2
+script:
+  - cd ansible && ansible-playbook --version
```

Comment:
[It](https://github.com/TheTorProject/ooni-sysadmin/commit/9b48318dd8029f39b5fb834ce7b8147ec9b8172c) works! https://travis-ci.org/TheTorProject/ooni-sysadmin/builds/257356900